Makefile

```diff
@@ -285,15 +285,51 @@ release:
 ###                     Local Mainnet Development                           ###
 ###############################################################################
 
-mainnet-zetarpc-node:
-	cd contrib/mainnet/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose up
-
-mainnet-bitcoind-node:
+start-bitcoin-node-mainnet:
 	cd contrib/mainnet/bitcoind && DOCKER_TAG=$(DOCKER_TAG) docker-compose up
 
-athens3-zetarpc-node:
+start-zetacored-rpc-mainnet:
+	cd contrib/mainnet/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose up
+
+start-zetacored-rpc-testnet:
 	cd contrib/athens3/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose up
 
+stop-bitcoin-node-mainnet:
+	cd contrib/mainnet/bitcoind && DOCKER_TAG=$(DOCKER_TAG) docker-compose down
+
+stop-zetacored-rpc-testnet:
+	cd contrib/athens3/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose down
+
+stop-zetacored-rpc-mainnet:
+	cd contrib/mainnet/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose down
+
+clean-bitcoin-node-mainnet:
+	cd contrib/mainnet/bitcoind && DOCKER_TAG=$(DOCKER_TAG) docker-compose down -v
+
+clean-zetacored-rpc-testnet:
+	cd contrib/athens3/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose down -v
+
+clean-zetacored-rpc-mainnet:
+	cd contrib/mainnet/zetacored && DOCKER_TAG=$(DOCKER_TAG) docker-compose down -v
+
+start-zetacored-rpc-mainnet-localbuild:
+	cd contrib/mainnet/zetacored-localbuild && docker-compose up --build
+
+start-zetacored-rpc-testnet-localbuild:
+	cd contrib/athens3/zetacored-localbuild && docker-compose up --build
+
+stop-zetacored-rpc-mainnet-localbuild:
+	cd contrib/mainnet/zetacored-localbuild && docker-compose down
+
+stop-zetacored-rpc-testnet-localbuild:
+	cd contrib/athens3/zetacored-localbuild && docker-compose down
+
+zetacored-rpc-mainnet-localbuild:
+	cd contrib/mainnet/zetacored-localbuild && docker-compose down -v
+
+zetacored-rpc-testnet-localbuild:
+	cd contrib/athens3/zetacored-localbuild && docker-compose down -v
+
```

Comment:
Should these be `stop-zetacored...`? Why do they start with no action but the others do?